core/lexicon/en/resource.inc.php

```diff
@@ -72,6 +72,7 @@
 $_lang['resource_err_new_parent_nf'] = 'New parent resource with id [[+id]] not found.';
 $_lang['resource_err_remove'] = 'An error occurred while trying to remove the resource.';
 $_lang['resource_err_save'] = 'An error occurred while trying to save the resource.';
+$_lang['resource_err_sort'] = 'An error occurred while trying to move a resource in the resource tree.';
```

Comment:
Could there be two error messages? One for not allowed and one for error occured?

Reply:
I think that we can think about this or rename the lexicon. It is necessary to understand in which case to display an error occured message ...